src/contracts/libraries/GPv2Encoding.sol

```diff
@@ -98,8 +111,8 @@ library GPv2Encoding {
             "GPv2: malformed order data"
         );
 
-        order.sellTokenIndex = uint8(encodedOrder[0]);
-        order.buyTokenIndex = uint8(encodedOrder[1]);
+        uint8 sellTokenIndex = uint8(encodedOrder[0]);
+        uint8 buyTokenIndex = uint8(encodedOrder[1]);
```

Comment:
Would it make sense to change the encoding sequence of the order so that you don't need a temporary variable for the sell and buy token ids?

Reply:
I'm not sure how. We need the `sellTokenIndex` for two fields, so keeping it on the stack just a microscopic gas optimization (although we can store in memory and then read it from memory right after).

Reply:
Ah, I failed to differentiate between memory and stack in my comment. There's nothing to do here! :heavy_check_mark: